### ISSUE TITLE
Module not found in fresh create-react-app

### ISSUE THREAD

**Issue by callil** [gm0t/react-sticky-el#9]
Can't seem to get this package to install correctly in a fresh create-react-app. All other packages (such as react-scroll) install correctly and can be found.

I'm attempting to import as directed by the readme:
`import Sticky from 'react-sticky-el';`

and get this error:
```
./src/App.js
Module not found: Can't resolve 'react-sticky-el' in ...

```
I've double checked its installation path and contents, deleted and re-installed node_modules etc.
Sorry for opening such a basic issue. Thanks.

**Comment by gm0t** [comment]
Hi! Yeah, there was a problem with build process.
It is now fixed in version 1.0.12, please update.

**Comment by callil** [comment]
The issue seems to be resolved.
Thanks for such a fast response!!
